src/utils/fetchLatestWaWebVersion.ts

```
@@ -0,0 +1,37 @@
+import axios, { AxiosRequestConfig } from 'axios';
+import { fetchLatestBaileysVersion, WAVersion } from 'baileys';
+
+export const fetchLatestWaWebVersion = async (options: AxiosRequestConfig<{}>) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)